pages.de/common/ssh-keygen.md

```diff
@@ -33,3 +29,7 @@
 - Ändere das Passwort eines privaten Schlüssels:
 
 `ssh-keygen -p -f ~/.ssh/{{datei}}`
+
+- Ändern Sie den Typ des Schlüssel formats (z. B. vom OPENSSH-Format in PEM), die Datei wird an Ort und Stelle neu geschrieben:
```

Comment:
```suggestion
- Ändere den Typ des Schlüsselformats (z. B. vom OPENSSH-Format in PEM), die Datei wird an Ort und Stelle neu geschrieben:
```
We generally don't use polite form in German pages.

Reply:
i used google translate for that 😄

Reply:
Oh, in that case it's not bad 😆